### PR TITLE
Update interest rate even if pool debt is 0

### PR DESCRIPTION
**Pull request by grandizzy**
- calculate mau only if pool debt is not 0
- always update pool interest rate, if pool debt is 0 then calculate with mau = 0

### CODE REVIEW

**Review by mattcushman — APPROVED** [review]
lgtm

File: src/libraries/external/PoolCommons.sol [hunk 1]
```diff
@@ -63,47 +74,45 @@ library PoolCommons {
...
-                )
-            );
+            // calculate meaningful actual utilization for interest rate update
+            mau    = int256(_utilization(deposits_, poolState_.debt, poolState_.collateral));
+            mau102 = mau * PERCENT_102 / 1e18;
+
+        }
 
-            int256 tu = (curDebtEma != 0 && curLupColEma != 0) ? int256(Maths.wdiv(curDebtEma, curLupColEma)) : int(Maths.WAD);
+        // update pool interest rate
```
Comment (MikeHathaway) [inline src/libraries/external/PoolCommons.sol]: nit: stray comment can be removed

**Review by MikeHathaway — APPROVED** [review]
minor question about comments, but LGTM

File: src/libraries/external/PoolCommons.sol (hunk 1 above)
Comment (grandizzy) [reply src/libraries/external/PoolCommons.sol]: removed with https://github.com/ajna-finance/contracts/pull/479/commits/ab06e0525d2a89f7e2b473719a582aa27c217ba6

**Pull request merged**